src/Serilog.Sinks.Loggly/Sinks/Loggly/FileSystemAdapter.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.IO;
+
+namespace Serilog.Sinks.Loggly
+{
+    /// <summary>
+    /// adapter to abstract away filesystem specific / coupled calls, especially using File and Directory
+    /// </summary>
+    internal class FileSystemAdapter : IFileSystemAdapter
```

Comment:
I think that we can use the new .NET Standard FileProvider instead of writing our own https://docs.microsoft.com/en-us/aspnet/core/api/microsoft.extensions.fileproviders.physicalfileprovider do you think it will fit?

Reply:
I'll look in to it. I currently just moved all File / Directory ops in use to here to enable testing.